casual/casual-caller/src/test/groovy/se/laz/casual/connection/caller/CasualCallerImplTest.groovy

```diff
@@ -24,6 +25,7 @@ class CasualCallerImplTest extends Specification
     CasualConnectionFactory fallBackConnectionFactory
     ConnectionFactoryEntry fallBackEntry
     TransactionLess transactionLess
+    HttpClient httpClient
```

Comment:
Where are the additional tests to check if the httpClient has been used or not? Additionally that tpCaller or transactionLess have not been called?
HttpClient should just be mocked here.
`0* httpClient._` etc etc